### ISSUE TITLE
[feat] - expose fan_hot_tolerance as a switch to enable/disable on frontend

### ISSUE THREAD

**Issue by Jeepmb** [swingerman/ha-dual-smart-thermostat#255]
TL;DR I'd like to turn on/off the fan_hot_tolerance feature (so only AC is running) more conveniently and with automations.

So I have been using the fan_hot_tolerance feature along with the fan_air_outside setting for a while now, and abolutely love it! The majority of the time, it performs exactly as I intend, and I don't have to think about it! It pulls in outside air when it's cool outside, and switches to AC only when it's not.

Recently though, wildfire season has started here, and we have really bad outdoor air quality due to smoke. In this case, it is undesirable for the fan to bring in outside air. The problem is with the current configuration I have to go into the config file, manually comment out the settings for fan_hot_tolerance, and restart home assistant whenever the outdoor air pollution gets bad. Otherwise the fan kicks on in the evening and fills the house with smoke.

My proposed solution is to expose an input boolean or switch in the HA frontend that allows you to enable/disable the fan_hot_tolerance feature altogether. I believe there are a number of use cases where this might be worth having a quick toggle for:

- In the pollution example, I can leverage my air quality and pollution integrations to create an automation that dynamically enables/disables the fan_hot_tolerance based on outdoor air quality
- If I see the rate-of-change of the indoor temperature rising above a certain rate (based on a derivative sensor I made), I can make an automation that temporarily disables the fan_hot_tolerance feature because I know I need greater cooling power more quickly than the outside air can provide.
- Disabling the fan_hot_tolerance feature to make the HVAC behave more predicably when friends/relatives are watching our house while we are away.

**Comment by swingerman** [comment]
Please check https://github.com/swingerman/ha-dual-smart-thermostat/releases/tag/v0.9.8.beta-4

use 
```yaml
fan_hot_tolerance_toggle: entity_id
```
To toggle the feature

**Comment by Jeepmb** [comment]
That was fast! Thanks for throwing that together so quickly. I tested it yesterday and found a bug. It seems if you toggle the fan_hot_tolerance_toggle entity, it will immediately turn on the cooling device (whether AC or fan depending on toggle state) regardless of the temperture or set point.
